root/static/scripts/edit/URLCleanup.js

```diff
@@ -915,6 +915,39 @@ const CLEANUPS: CleanupEntries = {
       return url.replace(/^(https:\/\/archive\.org\/details\/[A-Za-z0-9._-]+)\/$/, '$1');
     },
   },
+  'audiomack': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?audiomack\\.com/', 'i')],
+    restrict: [LINK_TYPES.streamingfree],
+    clean: function (url) {
+      url = url.replace(/^https?:\/\/(www.)?audiomack.com\//, 'https://audiomack.com/');
```

Comment:
Following previous comment, a bit of cleanup to remove potential trailing slash, `#comments` and whatnot:

```suggestion
      url = url.replace(/^https?:\/\/(www.)?audiomack.com\/([^\/?&#]+(?:\/(album|song)\/[^\/?&#]+)?).*$/, 'https://audiomack.com/$1');
```

Reply:
Without a capturing `(www)`, but this seems good, thanks!